### ISSUE TITLE
python2 unicode can no longer be copied to clipboard in v1.6.2

### ISSUE THREAD

**Issue by XN137** [asweigart/pyperclip#129]
due to https://github.com/asweigart/pyperclip/commit/fc1185ba0ead24c48d075ca80f1faeb072a89043
we now see this error:
```
yperclip.PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode
```

the commit seems to be included in v1.6.2 on pypi
but nothing about it is mentioned in https://github.com/asweigart/pyperclip/blob/master/CHANGES.txt (not even the version)

was this intentional? or is a check for `unicode` missing there?

**Comment by MattWatWork** [comment]
+1

**Comment by obskyr** [comment]
Very +1. A fix would make me happy!

**Comment by felixonmars** [comment]
The latest 1.6.4 version still has failed tests like this error:

```
============================= test session starts ==============================
platform linux2 -- Python 2.7.15, pytest-3.6.4, py-1.5.4, pluggy-0.7.1
rootdir: /build/python-pyperclip/src/pyperclip-1.6.4-py2, inifile:
collected 92 items

tests/test_pyperclip.py ssssssssssssssssssssssssssssssssssssssssssssssss [ 52%]
sssssssssssssss..FF..FF...FF..FF.sssssssss..                             [100%]

=================================== FAILURES ===================================
______________________ TestXClip.test_copy_paste_unicode _______________________

self = <test_pyperclip.TestXClip testMethod=test_copy_paste_unicode>

    def test_copy_paste_unicode(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
        msg = u"ಠ_ಠ"
>       self.copy(msg)

tests/test_pyperclip.py:72:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:195: in copy_xclip
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = 'ಠ_ಠ'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
___________________ TestXClip.test_copy_paste_unicode_emoji ____________________

self = <test_pyperclip.TestXClip testMethod=test_copy_paste_unicode_emoji>

    def test_copy_paste_unicode_emoji(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
        msg = u"🙆"
>       self.copy(msg)

tests/test_pyperclip.py:79:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:195: in copy_xclip
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = '🙆'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
_________________________ TestXClip.test_copy_unicode __________________________

self = <test_pyperclip.TestXClip testMethod=test_copy_unicode>

    def test_copy_unicode(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
>       self.copy(u"ಠ_ಠ")

tests/test_pyperclip.py:61:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:195: in copy_xclip
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = 'ಠ_ಠ'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
______________________ TestXClip.test_copy_unicode_emoji _______________________

self = <test_pyperclip.TestXClip testMethod=test_copy_unicode_emoji>

    def test_copy_unicode_emoji(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
>       self.copy(u"🙆")

tests/test_pyperclip.py:66:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:195: in copy_xclip
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = '🙆'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
_______________________ TestXSel.test_copy_paste_unicode _______________________

self = <test_pyperclip.TestXSel testMethod=test_copy_paste_unicode>

    def test_copy_paste_unicode(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
        msg = u"ಠ_ಠ"
>       self.copy(msg)

tests/test_pyperclip.py:72:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:223: in copy_xsel
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = 'ಠ_ಠ'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
____________________ TestXSel.test_copy_paste_unicode_emoji ____________________

self = <test_pyperclip.TestXSel testMethod=test_copy_paste_unicode_emoji>

    def test_copy_paste_unicode_emoji(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
        msg = u"🙆"
>       self.copy(msg)

tests/test_pyperclip.py:79:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:223: in copy_xsel
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = '🙆'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
__________________________ TestXSel.test_copy_unicode __________________________

self = <test_pyperclip.TestXSel testMethod=test_copy_unicode>

    def test_copy_unicode(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
>       self.copy(u"ಠ_ಠ")

tests/test_pyperclip.py:61:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:223: in copy_xsel
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = 'ಠ_ಠ'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
_______________________ TestXSel.test_copy_unicode_emoji _______________________

self = <test_pyperclip.TestXSel testMethod=test_copy_unicode_emoji>

    def test_copy_unicode_emoji(self):
        if not self.supports_unicode:
            raise unittest.SkipTest()
>       self.copy(u"🙆")

tests/test_pyperclip.py:66:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyperclip/__init__.py:223: in copy_xsel
    text = _stringifyText(text) # Converts non-str values to str.
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

text = '🙆'

    def _stringifyText(text):
        if not isinstance(text, (str, int, float, bool)):
>           raise PyperclipException('only str, int, float, and bool values can be copied to the clipboard, not %s' % (text.__class__.__name__))
E           PyperclipException: only str, int, float, and bool values can be copied to the clipboard, not unicode

pyperclip/__init__.py:99: PyperclipException
=============== 8 failed, 12 passed, 72 skipped in 0.81 seconds ================
```

**Comment by asweigart** [comment]
Fixed in 9283467
